R/aaa-odbc-data-type.R

```diff
@@ -15,61 +18,51 @@
 #' - BigQuery
 #' - Teradata
 #' - Access
+#' - Snowflake
+#'
+#' @details
 #'
 #' If you are using a different database and `dbWriteTable()` fails with a SQL
 #' parsing error the default method is not appropriate, you will need to write
-#' a new method.
-#'
-#' @section Defining a new dbDataType method:
-#'
-#' The object type for your connection will be the database name retrieved by
-#' `dbGetInfo(con)$dbms.name`. Use the documentation provided with your
-#' database to determine appropriate values for each R data type. An example
-#' method definition of a fictional `foo` database follows.
-#' ```
-#' con <- dbConnect(odbc::odbc(), "FooConnection")
-#' dbGetInfo(con)$dbms.name
-#' #> [1] "foo"
+#' a new method. The object type for your method will be the database name
+#' retrieved by `dbGetInfo(con)$dbms.name`. Use the documentation provided with
+#' your database to determine appropriate values for each R data type.
 #'
-#' `odbcDataType.foo <- function(con, obj, ...) {
-#'   switch_type(obj,
```

Comment:
This example used this unexported `switch_type()` helper. If we wanted to keep this example, we could export `switch_type()`, but us never having received an issue related to this makes me wonder if folks ever used this code.

Reply:
Yeah, I don't think there's a compelling reason to keep including this example.